projects/ccf-rui/src/app/core/store/icon-registry/icon-registry.state.ts

```diff
@@ -54,7 +54,7 @@ export class IconRegistryState extends NgxsDataRepository<void> {
     super();
 
     for (const { name, namespace, url, html } of DEFAULT_ICONS) {
-      const baseHref = globalConfig.baseHref || '';
+      const baseHref = globalConfig.embedded ? '' : globalConfig.baseHref || '';
```

Comment:
Again, why are you doing this???

Reply:
The organ won't display normally so I did this in order to test the PR. Forgot to revert it.